WordOfTheDay.Repository/Entities/WordContext.cs

```diff
@@ -7,6 +7,7 @@ public class WordContext : DbContext
         public WordContext (DbContextOptions<WordContext> options)
             : base(options)
         {
+            Database.EnsureCreated();
```

Comment:
I wouldn't use this. The reason is that usually a user that used for sql connection in production doesn't have permissions for creating databases